package.json

```diff
@@ -54,7 +54,7 @@
     "@radix-ui/react-tooltip": "0.1.1",
     "@radix-ui/react-use-layout-effect": "^0.1.0",
     "@radix-ui/react-visually-hidden": "0.1.3",
-    "@stitches/react": "1.2.6",
+    "@stitches/react": "1.2.7",
```

Comment:
:warning: There seems to be FOUC issues on 1.2.7:
- https://github.com/modulz/stitches/issues/995

Reply:
The issue is open by a random guy and is not confirmed + from the changes added In 1.2.7 if the issue is established it is probably already there before.

IMO we should block PR only if we actually found breaking changes or issues otherwise it will be tricky going forward since I'm pretty sure most of our dependencies have open issues 😅

Reply:
I am just sharing knowledge @matthieuh . I will review the PR when I have time. Thanks for your very friendly feedback :+1:

Reply:
What about testing a build of Faency with our NextJs project?